compliance_checker/suite.py

```diff
@@ -528,11 +543,38 @@ def is_binary_string(bts):
                                                   + list(range(0x20, 0x100))))
                 return bool(bts.translate(None, textchars))
 
+            def is_cdl_file(filename, data):
+                '''
+                Quick check for .cdl ascii file
+
+                Example:
+                    netcdf sample_file {
+                    dimensions:
+                        name_strlen = 7 ;
+                        time = 96 ;
+                    variables:
+                        float lat ;
+                            lat:units = "degrees_north" ;
+                            lat:standard_name = "latitude" ;
+                            lat:long_name = "station latitude" ;
+                    etc...
+
+                :param str filename: Absolute path of file to check
+                :param str data: First chuck of data from file to check
+                '''
+                if os.path.splitext(filename)[-1] == '.cdl':
+                    return True
+                if data.startswith('netcdf') or 'dimensions' in data:
```

Comment:
We'll leave it for now, we don't have to handle all the cases of CDL formatting. Just the ones where ncdump generates the CDL.